src/core/minigames/dropletBucket/dropletBucket.css

```
@@ -0,0 +1,149 @@
+#abcl-minigame.game-container {
+  position: relative;
+  height: 50em;
```

Stylelint (17.14.0)

[error] Duplicate property "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)